Not possible to compile / Serial Issue

Hej

I wasn't able to send any command through the serial terminal. I get a lot of feedback from the ESP32S about whats wrong with the voltages and so on. But it doesn't accept any commands.

So I've tried to build the project in PlatformIO. The available binary seems to be outdated anyways.
Unfortunately this is also not possible.
I get this message :
`In file included from lib/MPPTLib/solar.h:2,
                 from src/main.cpp:1:
lib/MPPTLib/publishable.h:3:10: fatal error: FreeRTOS.h: No such file or directory`

But regardless of which version of FreeRTOS I try, it will never compile.

I think this is a great project and I would really like to help with improving the documentation. I think it's missing a lot of things for people who are not so deep involved in programming but more in electronics. But first I need to get my system up and running. ;)

My Setup is a ESP32S with a DPS5020

--- a/lib/MPPTLib/publishable.h
+++ b/lib/MPPTLib/publishable.h
@@ -1,6 +1,7 @@
 #pragma once
 #include <WString.h>
-#include <FreeRTOS.h>
+#include <freertos/FreeRTOS.h>
+#include <freertos/semphr.h>
 #include <functional>
 #include <map>
 #include <list>

